src/inventory.cpp

```diff
@@ -134,7 +117,6 @@ void ItemStack::deSerialize(std::istream &is, IItemDefManager *itemdef)
 			throw SerializationError("Too large material number");
 		// Convert old id to name
```

Comment:
So, you think that the whole `else if(name == "MaterialItem2")` branch can be removed?